firmware/controllers/sensors/converters/vehicle_speed_converter.h

```diff
@@ -0,0 +1,17 @@
+#include "pch.h"
+#include "sensor_converter_func.h"
+
+class VehicleSpeedConverter : public SensorConverter {
+public:
+	DECLARE_ENGINE_PTR;
+
+	VehicleSpeedConverter() {
```

Comment:
no need for empty constructor, this is what c++ will generate by default

Reply:
Noted